services/meta/ota.md

```diff
@@ -7,34 +7,44 @@ This service is used to manage over-the-air (OTA) upgrades of end devices.
 
 #### Interfaces
 
-Type | Interface               | Value type | Description
------|-------------------------|------------|--------------
-out  | evt.ota_progress.report | int_map    | Shows upgrade progress and estimated time left.
-out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade status.
+Type | Interface               | Value type | Description                                                                                                      |
+-----|-------------------------|------------|------------------------------------------------------------------------------------------------------------------|
+in   | cmd.status.get_report   | null       | Requests update progress information.                                                                            |
+out  | evt.status.report       | object     | Shows update progress and estimated time left. See [`ota_report_object`](#definitions) for detailed information. |
+in   | cmd.update.install      | string     | Starts an update of the device. Value should be the full path to the update file.                                | 
 
+## Definitions
+
+* `ota_report_object` is an object with the following structure:
+
+| Field          | Type   | Example         | Description                                                                                                     |
+|----------------|--------|-----------------|-----------------------------------------------------------------------------------------------------------------|
+| state          | string | `"IN PROGRESS"` | One of the `state` values.                                                                                      |
+| substate       | string | `"TRANSFERING"` | One of the `substate` values.                                                                                   |       
+| progress       | int    | `10`            | Optional, only available in substate `TRANSFERING`. Progress percentage of the update transfer.                 |
+| remaining_min  | int    | `3`             | Optional, only available in substate `TRANSFERING`. Remaining time until the end of the transfer (minutes).     | 
+| remaining_sec  | int    | `35`            | Optional, only available in substate `TRANSFERING`. Remaining time until the end of the transfer (seconds).     |
+| last_update    | int    | `132351314`     | Optional. Unix timestamp of the last started update process.                                                    |        
+
+
+* `state`: Current state of the update process, can be one of: `"NOT STARTED"`, `"IN PROGRESS"`, `"FINISHED"`.
```

Comment:
NOT_STARTED -> IDLE

Reply:
NOT_STARTED is not idle IMHO, it means never_started, as after first OTA update the adapter should always return FINISHED or COMPLETED.